packages/venia-concept/src/components/Checkbox/checkbox.js

```diff
@@ -1,90 +1,62 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-import defaultClasses from './checkbox.css';
+import React, { Component, Fragment } from 'react';
+import { bool, node, shape, string } from 'prop-types';
+import { BasicCheckbox, asField } from 'informed';
+import { compose } from 'redux';
+
 import classify from 'src/classify';
 import Icon from 'src/components/Icon';
+import defaultClasses from './checkbox.css';
 
-class Checkbox extends Component {
```

Comment:
This is the only field that had an existing Venia component that I had to change. For `Input`, rather than rewriting it, I made a new component called `TextInput`.